packages/ecc-client-ga4gh-wes/src/components/wes-run/definition/workflowData.d.ts

```diff
@@ -0,0 +1,35 @@
+interface Tags {
+  [key: string]: string;
+}
+
+interface WorkflowEngineParameters {
+  [key: string]: string;
+}
+
+interface RunLog {
+  name: string;
+  cmd: string[];
+  start_time: string;
+  end_time: string;
+  stdout: string;
+  stderr: string;
+  exit_code: number;
+}
+
+type TaskLog = RunLog;
```

Comment:
what's going on here?

Reply:
API structure, in future if changes are made, dev understands those two are separate entity.

Reply:
let them use the same interface, if there are any changes I'm pretty sure anyone that takes a look at the code can easily understand and make modifications.

Reply:
The file is removed; please resolve.